### ISSUE TITLE
Numeric confusion matrix does not match with percentage confusion matrix

### ISSUE THREAD

**Issue by kilickursat** [pycaret/pycaret#2340]
I have worked with imbalanced datasets and created confusing matrix. The numeric based CM shows the distribution of the test data properly. However, the percentage CM is not matching with numeric CM, so what is the reason for it? 

`session_1 = setup(df2, target = 'Layers', session_id=123, log_experiment=False, experiment_name='lithologies', normalize_method="minmax",fix_imbalance=False,remove_multicollinearity = True, multicollinearity_threshold = 0.6)`

![Screenshot 2022-03-29 165211](https://user-images.githubusercontent.com/25801054/160561526-c7756588-1ef4-4151-ace3-d700e9dcbb67.png)

**Numeric CM**
![CM With numeric](https://user-images.githubusercontent.com/25801054/160561096-036f0458-cac6-4dac-b5a3-2f0877c780d4.png)

**Percentage CM**

![lightgbm cm](https://user-images.githubusercontent.com/25801054/160561184-82449fda-c5a4-4d93-9d72-332a6294da5c.png)

145 is not 94% of the 196 test data.

**Comment by moezali1** [comment]
@kilickursat This is really strange. Thanks for bringing this up. We are actually using [yellowbrick](https://github.com/DistrictDataLabs/yellowbrick) project for all the visualizations. Can you check to see if there is some parameter you are missing or the way multi-class confusion matrix percentages are calculated is different or is it really a bug in [yellowbrick](https://github.com/DistrictDataLabs/yellowbrick)? 

I found results for binary class classification to be okay. Thanks for your help.

**Comment by kilickursat** [comment]
Thanks for your reply. I will apply and share here my results. However, I realized that I applied training and test with Fold Number together, so I thought the reason for it was because of the two splitting methods. But I could not remove the Fold Number from set-up, how can I remove fold number from setup?

**Comment by kilickursat** [comment]
In addition, it might show different metrics like AUC instead of Accuracy?

**Comment by kilickursat** [comment]
About the dataset, as you can see there is no missing value in the model summary. I have 18 features and 3 classes but I applied multicollinearity.  I applied yellowbrick with logicregression the results:

**Yellowbrick CM**
![github LR](https://user-images.githubusercontent.com/25801054/160591354-988ba7f8-2b03-4d4f-9429-fa7097112054.png)

**Pycaret CM**

![pycaret cm](https://user-images.githubusercontent.com/25801054/160591448-f38e8464-adc1-4d7c-9350-2e98a6d41114.png)

**Pycaret CM percentage**
![pycaret cm2](https://user-images.githubusercontent.com/25801054/160591498-deb100da-7194-482b-9b21-3d329ae1029a.png)

**Comment by kilickursat** [comment]
> multi-class confusion matrix percentages are calculated in different

Hi, @moezali1, I could not solve the problem. How can I check the calculation way of the confusion matrix? Most probably different metrics work with CM, the result is not accuracy based.

**Comment by moezali1** [comment]
Closing as part of the 4.0 backlog cleanup — see #4177.

PyCaret 4.0 is a ground-up rewrite (OOP-only API, sklearn-native engine, Python 3.11+, sklearn ≥ 1.7) and almost all pre-4.0 issues either reference removed APIs (`setup()`, `plot_model`, `evaluate_model`, `interpret_model`, `automl`, the legacy time-series module, the Streamlit dashboard, …) or no longer reproduce on the new code path.

**If this still affects you in 4.0, please re-file as a new issue** with:

- the experiment constructor call (`Experiment(...)`),
- the operation that fails,
- the full traceback,
- your Python / sklearn / pycaret versions.

Resources for 4.0:

- Docs: https://pycaret.org/docs
- Migrate from 3.x: https://pycaret.org/docs/guides/migrating-from-3-x
- API reference: https://pycaret.org/reference
- FAQ: https://pycaret.org/docs/resources/faqs

Thanks for filing — and apologies for the long latency. The repo had drifted; we're starting clean.
